ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__ocp-4.10-upgrade-from-stable-4.9.yaml

```diff
@@ -25,7 +20,17 @@ tests:
     cluster_profile: aws
     dependencies:
       OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:initial
-    workflow: cucushift-aws-ipi-upgrade
+    env:
+      UPGRADE_CHECK_RUN_TAGS: '@aws-ipi and @4.10 and @upgrade-check'
+      UPGRADE_PRE_RUN_TAGS: '@aws-ipi and @4.9 and @upgrade-prepare'
+      UPGRADE_SKIP_TAGS: not @flaky and not @inactive and not @stage-only and not
+        @prod-only and not @qeci and not @proxy and not @disconnected and not @console
+        and not @destructive
```

Comment:
No need this env here as it is the same with the default values.

Reply:
OK, I was trying to make these more explicit to make overriding the config easier. I'm ok to remove this here given the value is the same as default.

Reply:
The idea is that people will copy/paste that blindly, and we have to maintain the same tag collections in several places then.